.travis.yml

```diff
@@ -0,0 +1,24 @@
+# On Travis set: SONATYPE_USERNAME, SONATYPE_PASSWORD, GPG_NAME, GPG_EMAIL
+
+language: java
+jdk: oraclejdk8
+
+cache:
+  directories:
+  - $HOME/.m2
+
+before_install: source .travis.gpg.sh
+
+install: mvn install -settings .travis.settings.xml -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
Should this be `mvn install --settings ...` ... double dash?

Reply:
That's strange, I guess it's not required for the install command.  I'll create a new PR to fix.